go/ssl/ssl.go

```diff
@@ -46,8 +33,8 @@ func NewTLSConfig(caFile string, verifyCert bool) (*tls.Config, error) {
 
 	// Set to TLS 1.2 as a minimum.  This is overridden for mysql communication
 	c.MinVersion = tls.VersionTLS12
-	// Remove insecure ciphers from the list
-	c.CipherSuites = cipherSuites
+	// "If CipherSuites is nil, a default list of secure cipher suites is used"
```

Comment:
What is the default list of secure cipher suites?

Reply:
It's roughly https://golang.org/src/crypto/tls/cipher_suites.go?s=1340:1374#L40
(The ordering can vary depending on what hardware features are available)

Reply:
Thank you